style.css

```diff
@@ -1,3 +1,5 @@
+@import "./utils.css";
```

Comment:
We shouldn't have this `utils.css`. Remove it.

Also, remove the file, it should not appear under `files changed` here in GitHub